klippy/extras/virtual_sdcard.py

```diff
@@ -124,8 +128,9 @@ def cmd_M26(self, params):
     def cmd_M27(self, params):
         # Report SD print status
         if self.current_file is None or self.work_timer is None:
-            self.gcode.respond("Not SD printing.")
-            return
+            if not self.must_pause_work:
+                self.gcode.respond("Not SD printing.")
+                return
```

Comment:
This is the patch that resolves the octoprint issue.  It adds a check to make sure it isn't paused before responding with "Not SD printing" and returning.  Like above, I can replace must_pause_work with a member variable that specifically tracks when virtual_sd is paused.